src/app/api/dashboard_service/dashboard.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
+
+import { UserClass } from '../Classes/user';
+import { WeatherClass } from '../Classes/weather';
+import { LoginService } from '../login_service/login.service';
+import { HeaderService } from '../header_service/header.service';
+
+@Injectable()
+export class DashboardService {
```

Comment:
Don't name services by place where they are used, but by their functionality (YahooApiService, TemperatureConersionService, AuthenticationService)

- [ ] rename service

Reply:
Services in my case are not organized in a good way, so they have multiple functionalities and the name is not obvious. Do their names need to start with capital letter?

Reply:
Yeah services are singletons and Classes both of those reasons give them right to be named with capital letter